run_release.py

```diff
@@ -100,7 +100,7 @@
    so it's worth checking back even after reading earlier versions.
 
 
-Summary -- Release highlights
+Summary -- release highlights
 =============================
```

Comment:
```suggestion
Summary --- release highlights
==============================
```
Given the dash, this could also work with an uppercase `R`, but up to you:
```suggestion
Summary --- Release highlights
==============================
```